provider/core.rb

```diff
@@ -63,14 +63,17 @@ def copy_files(output_folder)
       copy_file_list(output_folder, @config.files.copy)
     end
 
-    def copy_common_files(output_folder, _version_name = nil)
+    # version_name is used within templates because binding is passed down
```

Comment:
what do you mean by "because binding is passed down"?

Reply:
Update comment, hopefully making it a bit more clear :)